src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,170 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final String FORMAT = "%s is %s %s";
+    private static final String PATTERN_OFFSET = "dd MMMM yyyy HH:mm";
```

Comment:
```suggestion
    private static final String PATTERN_FULL = "dd MMMM yyyy HH:mm";
```
IMHO, there is no OFFSET in this pattern